docs/DeveloperGuide.md

```diff
@@ -404,6 +404,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 --------------------------------------------------------------------------------------------------------------------
 
+## **Appendix: Planned Enhancement**
+
+1. Currently, if any component (name, email, etc.) is too long, it will be hidden on the screen. We plan to implement
+   appropriate mechanism to show the full component.
+2. Currently, names like "`John Doe`" and "`John  Doe`" (two spaces in the middle) are considered different. We plan
+   to have more sophisticated duplication detection mechanism.
+3. Currently, `undo` only supports undoing one command. We plan to supports undoing multiple commands, as well as
+   implementing a `redo` command.
```

Comment:
> each item in the list describing exactly one planned enhancement.

Since we already described undoing multiple commands as enhancement earlier, best to list this as a stand-alone point.